src/pages/tutorials/library.tsx

```diff
@@ -16,12 +16,28 @@ import {
 import SidebarNavList from 'components/sidebar/components/sidebar-nav-list'
 import { SidebarNavMenuItem } from 'components/sidebar/components'
 import { generateTopLevelSubNavItems } from 'lib/generate-top-level-sub-nav-items'
+import { getTutorials } from 'lib/learn-client/api/tutorial'
+import { Tutorial } from 'lib/learn-client/types'
+import { stripUndefinedProperties } from 'lib/strip-undefined-props'
+
+const DEFAULT_SLUGS = [
+	'terraform/infrastructure-as-code',
+	'vault/getting-started-intro',
+	'consul/get-started',
+	'nomad/get-started-install',
+	'vagrant/getting-started-index',
+	'packer/get-started-install-cli',
+	'boundary/getting-started-intro',
+	'waypoint/get-started-intro',
+	'cloud/terraform-hcp-provider-vault',
+]
```

Comment:
Sourced from the current learn list.

Reply:
My ocd is like 'can we make it 9' to complete the grid 😆